app.js

```diff
@@ -10,11 +10,71 @@ app.use(require('body-parser').json())
 const data = {
   fruits: [],
   vegetables: []
+  //fruits: [{ "id": "14r5y6", "name": "apple", "price": "0.49" }],
+  //vegetables: [{ "id": "y7474u", "name": "cilantro", "price": "1.00" },
+  //{ "id": "8ue38u", "name": "onion", "price": "2.00" }]
 }
 
+// IE compatibility polyfill for string.prototype.includes()
+// Sunho Hong https://stackoverflow.com/a/39744409
+if (!Array.prototype.includes) {
+  Object.defineProperty(Array.prototype, "includes", {
+    enumerable: false,
+    value: function(obj) {
+        var newArr = this.filter(function(el) {
+          return el == obj;
+        });
+        return newArr.length > 0;
+      }
+  });
+}
+// IE compatibility pollyfill for string.prototype.filter
+//http://independent-software.com/extending-the-javascript-array-prototype-with-polyfills.html
+if (!Array.prototype.filter) {
+  Array.prototype.filter = function (callbackfn, /*optional*/ thisArg) {
+    var k, len, result = [];
+    
+    // Method cannot be run on an array that does not exist.
+    if (this == null) {
+      throw new TypeError('this is null or not defined');
+    }
+    
+    // The callback must be a function.
+    if (typeof callbackfn !== 'function') {
+      throw new TypeError();
+    }
+    
+    // Loop through array.
+    len = this.length;
+    k = 0;
+    while (k < len) {
+      if (k in this) {
+        // For each element, if callback returns truthy, add it to
+        // result array.
+        if (callbackfn.call(thisArg, this[k], k, this)) {
+          result.push(this[k]);
+        }
+      }
+      k = k + 1;
+    }
+    return result;
+  };
+}
```

Comment:
Both of these should not be necessary. Were you having issues calling either of these functions?